framework/src/main/java/org/checkerframework/framework/util/ContractsUtils.java

```diff
@@ -230,25 +242,38 @@ public int hashCode() {
         }
     }
 
-    public List<Contract> getContracts(ExecutableElement element) {
+    /**
+     * Returns all the contracts on {@code methodElement}.
+     *
+     * @param methodElement the method or constructor whose contracts to retrieve
+     * @return the contracts on {@code methodElement}
+     */
+    public List<Contract> getContracts(ExecutableElement methodElement) {
         List<Contract> contracts = new ArrayList<>();
-        contracts.addAll(getPreconditions(element));
-        contracts.addAll(getPostconditions(element));
-        contracts.addAll(getConditionalPostconditions(element));
+        contracts.addAll(getPreconditions(methodElement));
+        contracts.addAll(getPostconditions(methodElement));
+        contracts.addAll(getConditionalPostconditions(methodElement));
         return contracts;
     }
 
-    /** Returns the set of preconditions on the element {@code element}. */
-    public Set<Precondition> getPreconditions(Element element) {
+    /// Precondition methods (keep in sync with other two types)
+
+    /**
+     * Returns the preconditions on {@code methodElement}.
+     *
+     * @param methodElement the method or contract whose preconditions to return
```

Comment:
Should the `contract` be `constructor`?
Same comment about the naming of the parameter.

Reply:
Same for other methods later.

Reply:
I noticed this and fixed it recently.